src/azul/project/hca/transformers.py

```diff
@@ -744,7 +828,20 @@ def transform(self,
                         donors=list(map(self._donor, visitor.donors.values())),
                         organoids=list(map(self._organoid, visitor.organoids.values())),
                         files=list(map(self._file, visitor.files.values())),
-                        protocols=list(map(self._protocol, visitor.protocols.values())),
+                        analysis_protocols=list(
```

Comment:
Nit. this section for building the `contents` occurs multiple times, seems like it could be broken down into a function of a parent class.

Reply:
Good idea. I believe the `contents` section of the Transformers hasn't been factored out before due to the variety of which inner entities are singletons for each of the different Transformers, however since these protocols inner entities are constructed the same in all the Transformers and they have an inherent grouping (all being a protocol) I like the idea of factoring them out to remove the code duplication.